ethpm/assets/test_greeter.py

```diff
@@ -1,15 +0,0 @@
-def test_greeter(chain):
```

Comment:
I just deleted this file, since it didn't appear to be of much importance - but I could be wrong?

Reply:
probably leftover from a copy/paste between projects.